crates/sui-core/src/consensus_handler.rs

```diff
@@ -33,6 +39,10 @@ pub struct ConsensusHandler<T> {
     transaction_manager: Arc<TransactionManager>,
     /// parent_sync_store is needed when determining the next version to assign for shared objects.
     parent_sync_store: T,
+    /// Reputation scores used by consensus adapter that we update, forwarded from consensus
+    low_scoring_authorities: ArcSwap<HashMap<AuthorityName, u64>>,
```

Comment:
I would still recommend hiding all this behind a service implementation - or as @andll suggested maybe behind the `ConnectionMonitorStatus` (rebranded as something that can include this scope as well).